python/cudaq/kernel/ast_bridge.py

```diff
@@ -2461,8 +2645,7 @@ def compile_to_mlir(astModule, **kwargs):
     try:
         pm.run(bridge.module)
     except:
-        print(bridge.module)
-        raise RuntimeError("could not canonicalize code for '{}'.".format(
+        raise RuntimeError("could not compile code for '{}'.".format(
             bridge.name))
```

Comment:
Refactoring opportunity - consistent wording across error messages, e.g. capitalization and tool name. Can be done later.